components/Dropdown/Dropdown.js

```diff
@@ -1,27 +1,24 @@
 class Dropdown {
   constructor(element) {
-    
     // Assign this.element to the dropdown element
-    this.element;
-    
+    this.element = element;
+  
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = this.element.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = this.element.querySelector('.dropdown-content');
     
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    this.button.addEventListener('click', this.toggleContent.bind(this));
+    //this.button.addEventListener('click', () => this.toggleContent());
   }
-
   toggleContent() {
-    
     // Toggle the ".dropdown-hidden" class off and on
-    this.content;
+    this.content.classList.toggle('dropdown-hidden');
+    //console.log('clicked dropdown!');
   }
-}
+} // no errors, but the dropdown isn't working. 
 
 
```

Comment:
Since the issue with you dropdown menu has already been resolved, this comment can be safely removed (also to avoid confusion).